src/routes/Send.tsx

```diff
@@ -635,15 +660,12 @@ export default function Send() {
                     >
                         <Switch>
                             <Match when={sentDetails()?.failure_reason}>
-                                <MegaEx />
-                                <h1 class="mb-2 mt-4 w-full text-center text-2xl font-semibold md:text-3xl">
-                                    {sentDetails()?.amount
-                                        ? source() === "onchain"
-                                            ? i18n.t("send.payment_initiated")
-                                            : i18n.t("send.payment_sent")
-                                        : sentDetails()?.failure_reason}
-                                </h1>
-                                {/*TODO: add failure hint logic for different failure conditions*/}
```

Comment:
looks like you removed the on-chain success, but i dont know if this would ever be hit inside of the current `Match`. Also I see the same code below anyways

Reply:
yeah my theory is that was just bad copy / pasting